dev-tools/generate_notice.py

```diff
@@ -10,7 +11,7 @@ def read_file(filename):
         print("File not found {}".format(filename))
         return ""
 
-    with open(filename, 'r') as f:
+    with open(filename, 'rb') as f:
```

Comment:
Just FYI, I went for a different solution in https://github.com/elastic/beats/pull/4383, to avoid all the `six.b` calls.